src/migrations/20190920_ADDED_REFRESH_TOKEN.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex, Promise) {
+  return knex.schema.table('users', function (t) {
+    t.string('refreshToken').notNull().defaultTo('');
```

Comment:
Keeping the default value as `''` doesn't make sense here. You should just make the column nullable.

Also, keep in mind that if you store refresh token in the `users` table, that means one user only have a single login session as a new login would probably update this column and previous refresh tokens would be lost.
Is this the functionality you are trying to achieve?

Reply:
ok dai ma yo cahi banaunchu

Reply:
ani dai harek device chinna lai cahi k unique identifier pathaune device bata? mac address le huncha?